idaes/models_extra/rare_earth_recovery/ree_initial_flowsheet.py

```diff
@@ -0,0 +1,106 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory,
+# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
+# University, West Virginia University Research Corporation, et al.
+# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
+# for full copyright and license information.
+#################################################################################
+
+__author__ = "Costing Team (B. Paul and M. Zamarripa)"
+__version__ = "1.0.0"
+
+import pyomo.environ as pyo
```

Comment:
Again, is this the right place for this? For one, it is rather isolated at the moment, and it appears to be more of a minimal test case for costing than an actual flowsheet (thus the name would probably be misleading to users). Would this be better placed in the PrOMMiS workspace, which is where the first process flowsheet will go as well (at least until we have something more comprehensive to show to the world)?

Reply:
https://github.com/prommis/workspace